templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -2,6 +2,67 @@
 	<div class="ui segment metas">
 		{{template "repo/issue/branch_selector_field" .}}
 
+		{{if .Issue.IsPull }}
+
+		<input id="reviewer_id" name="reviewer_id" type="hidden" value="{{.reviewer_id}}">
+		<div class="ui {{if or (not .IsIssueWriter) .Repository.IsArchived}}disabled{{end}} floating jump select-reviewers-modify dropdown">
+			<span class="text">
+				<strong>{{.i18n.Tr "repo.issues.review.reviewers"}}</strong>
+				{{if and .IsIssueWriter (not .Repository.IsArchived)}}
+					{{svg "octicon-gear" 16}}
+				{{end}}
+			</span>
+			<div class="filter menu" data-action="" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/request_review">
+				{{range .Assignees}}
```

Comment:
Is `.Assignees` correct here?

Reply:
Because assignees  include the users who have wrie permission, that is same as I looking forward , so just reuse it but remove the poster of this PR.

Reply:
There's a difference between what users have write PR permissions and what users are able to give reviews. Who can review (and whose reviews are considered official) is calculated according to branch protection settings. If the target branch is not protected, all users with read access to the PR can review and all reviews are "official".

Users with write PR permissions are meant to perform administrative tasks on them, like closing, editing the title, etc. They may not be able to submit reviews! (except the site or repo admins, which I believe always can).

Reply:
I see,That mean we should allow  all reader can be requested to review except heself and the poster, But If it's a public repository , It will mean ... :cry:

Reply:
may be it shold beaome a search instead of a list

Reply:
Yes, I think so too.